src/__tests__/Cities.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
- [ ] Please fix this error as the previous reviewer requested:
![image](https://github.com/mohamedgamaleldin999999/metrics-app/assets/91486791/a2f724b0-4e07-45a3-9f15-589ea6c361f0)